File: PHP/Account.php
<?php
//Clase cuenta
class Account
{
    // Atributos o propiedades de la clase
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;

    //Método constructor
    public function __construct($name, $document)
    {
        $this->name = $name;
        $this->document = $document;
    }
}

File: Python/account.py
# Clase cuenta
class Account:

    # Atributos o propiedades de la clase
    id = int
    name = str
    document = str
    email = str
    password = str

    # Método constructor
    def __init__(self, name, document):
        self.name = name
        self.document = document
